.github/workflows/daily.yml

```diff
@@ -731,20 +774,20 @@ jobs:
         echo "skiptests: ${{github.event.inputs.skiptests}}"
         echo "test_args: ${{github.event.inputs.test_args}}"
         echo "cluster_test_args: ${{github.event.inputs.cluster_test_args}}"
-    # On centos7 actions/checkout@v4 does not work, so we use v3
-    # ref. https://github.com/actions/checkout/issues/1487
-    - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
+    - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
       with:
         repository: ${{ env.GITHUB_REPOSITORY }}
         ref: ${{ env.GITHUB_HEAD_REF }}
+    - name: Install EPEL
+      if: matrix.install_epel
+      run: dnf -y install epel-release
     - name: make
       run: |
-        yum -y install centos-release-scl epel-release
-        yum -y install devtoolset-7 openssl-devel openssl
-        scl enable devtoolset-7 "make BUILD_TLS=module SERVER_CFLAGS='-Werror'"
+        dnf -y install make gcc openssl-devel openssl procps-ng which /usr/bin/kill
```

Comment:
Is `/usr/bin/kill` supposed to be here?

Reply:
Yes, it's a valid way to install an RPM, and between alma/fedora versions the package names vary a little.

Reply:
Should it be adjusted above as well?

https://github.com/valkey-io/valkey/pull/543/files#diff-49aca37051e035bf25defdedef1928d2d6fa0e39d4a592891a8c74f971992f57R722

Reply:
Good catch, fixed to be consistent.